codex-ui/dev/Playground.vue

```diff
@@ -322,6 +327,43 @@ const formSectionItems = [
   { id: 2,
     name: 'Nickmel' },
 ];
+
+/**
+ * Items for searching in the context menu
+ */
+const contextMenuItems: ContextMenuItems[] = [
+  {
+    type: 'default',
```

Comment:
can we omit type when it is default?

Reply:
What do you mean?

Reply:
I mean to treat item as "default" if type is not specified